services/scribe/backfill/contract_test.go

```diff
@@ -248,86 +249,88 @@ func (b BackfillSuite) TestContractBackfill() {
 	Equal(b.T(), txBlockNumber, lastIndexed)
 }
 
-// TestTxTypeNotSupported tests how the contract backfiller handles a transaction type that is not supported.
+// // TestTxTypeNotSupported tests how the contract backfiller handles a transaction type that is not supported.
+// //
+// // nolint:dupl
 //
-// nolint:dupl
-func (b BackfillSuite) TestTxTypeNotSupported() {
-	if os.Getenv("CI") != "" {
-		b.T().Skip("Network test flake")
-	}
-
-	var backendClient backfill.ScribeBackend
-	omnirpcURL := "https://rpc.interoperability.institute/confirmations/1/rpc/42161"
-	backendClient, err := backfill.DialBackend(b.GetTestContext(), omnirpcURL, b.metrics)
-	Nil(b.T(), err)
-
-	// This config is using this block https://arbiscan.io/block/6262099
-	// and this tx https://arbiscan.io/tx/0x8800222adf9578fb576db0bd7fb4860fe89932549be084a3313939c03e4d279d
-	// with a unique Arbitrum type to verify that anomalous tx type is handled correctly.
-	contractConfig := config.ContractConfig{
-		Address:    "0xA67b7147DcE20D6F25Fd9ABfBCB1c3cA74E11f0B",
-		StartBlock: 6262099,
-	}
-
-	chainConfig := config.ChainConfig{
-		ChainID:               42161,
-		RequiredConfirmations: 0,
-		Contracts:             []config.ContractConfig{contractConfig},
-	}
-	backendClientArr := []backfill.ScribeBackend{backendClient, backendClient}
-	chainBackfiller, err := backfill.NewChainBackfiller(b.testDB, backendClientArr, chainConfig, 1, b.metrics)
-	Nil(b.T(), err)
-	err = chainBackfiller.Backfill(b.GetTestContext(), &contractConfig.StartBlock, false)
-	Nil(b.T(), err)
-
-	logs, err := b.testDB.RetrieveLogsWithFilter(b.GetTestContext(), db.LogFilter{}, 1)
-	Nil(b.T(), err)
-	Equal(b.T(), 4, len(logs))
-	receipts, err := b.testDB.RetrieveReceiptsWithFilter(b.GetTestContext(), db.ReceiptFilter{}, 1)
-	Nil(b.T(), err)
-	Equal(b.T(), 1, len(receipts))
-}
-
-// TestTxTypeNotSupported tests how the contract backfiller handles a transaction type that is not supported.
+//	func (b BackfillSuite) TestTxTypeNotSupported() {
```

Comment:
We shouldn't be adding less tests here. If anything, a lot of this codebase is lacking tests around edge cases 

Maybe we can fetch these txes as json from the rpcs and embed them as constants in our tests so we don't need to do the network fetches?

Reply:
was commented out so i could run new tests faster locally, will un comment pre-merge